roles/openshift_facts/tasks/main.yml

```diff
@@ -1,11 +1,10 @@
 ---
-- name: Verify Ansible version is greater than 1.8.0 and not 1.9.0 and not 1.9.0.1
-  assert:
-    that:
-    - ansible_version | version_compare('1.8.0', 'ge')
-    - ansible_version | version_compare('1.9.0', 'ne')
-    - ansible_version | version_compare('1.9.0.1', 'ne')
-    
+- name: Verify Ansible version is greater than or equal to 1.9.4 and less than 2.0
+  fail:
+    when:
+    - ansible_version | version_compare('1.9.4', 'lt')
```

Comment:
@abutcher The assert to fail change probably makes the diff a bit more confusing that it should be.

basically trying to enforce version 1.9.4 or greater but less than 2.0, so only valid match currently is 1.9.4 hence the less than.